src/vs/platform/actions/browser/menuEntryActionViewItem.ts

```diff
@@ -333,13 +335,13 @@ export class LabeledMenuItemActionItem extends MenuEntryActionViewItem {
 				iconClass = MenuEntryActionViewItem.ICON_PATH_TO_CSS_RULES.get(iconPathMapKey)!;
 			} else {
 				iconClass = ids.nextId();
-				createCSSRule(`.icon.${iconClass}`, `background-image: ${asCSSUrl(item.icon.light || item.icon.dark)}`);
-				createCSSRule(`.vs-dark .icon.${iconClass}, .hc-black .icon.${iconClass}`, `background-image: ${asCSSUrl(item.icon.dark)}`);
+				createCSSRule(`.codicon.${iconClass}`, `background-image: ${asCSSUrl(item.icon.light || item.icon.dark)}`);
```

Comment:
I was suggesting that you modify the LabeledMenuItemActionItem class since that's a custom class we added ourselves and is what's used for the action bar. The less we modify core VS code the better. 

I would also revert the change to the ids unless it's actually necessary - again so that we reduce the number of conflicting changes we make to core VS code.

Reply:
Changed the id back to the old one.